src/server/scripts/Commands/cs_debug.cpp

```diff
@@ -491,21 +436,13 @@ class debug_commandscript : public CommandScript
         return true;
     }
 
-    static bool HandleDebugUpdateWorldStateCommand(ChatHandler* handler, char const* args)
+    static bool HandleDebugUpdateWorldStateCommand(ChatHandler* handler, uint32 variable, uint32 value)
```

Comment:
Actually it should be `variableId` since that's how the packet field is named. So rather keep like it is or go for `variableId`.

https://github.com/TrinityCore/TrinityCore/blob/a9a4f05a282fd85e4f22923ec251c92e79581a47/src/server/game/Server/Packets/WorldStatePackets.h#L56

Reply:
hm, never looked at the packet really

fine by me